meetings/2021-11-15.md

```diff
@@ -0,0 +1,125 @@
+  * ...: previously the access token would have the `client_id` value, but we will need a clear mechanism to identify the client
+  * ...: in the simple version the OP would be responsible for verifying the identity of the client
+* AC: client authentication seems to be folded in interaction with OP
+* eP: for client_credentials, all the interactions happen via the OP, so the AS will rely on what the OP asserts.
+  * ...: we will want to ensure that the client identity is included in the ID token and eventual access token from the AS
+
+ACTION: eP will add the Client Identifier flow will be integrated into this flow
+
+
+### Test Suite
+
+https://github.com/solid/solid-oidc-tests
+https://solid.github.io/solid-oidc-tests/
+
+* AC: There is repository https://github.com/solid/solid-oidc-tests
+  * ...: It uses github pages to produce a static app
+  * ...: I can, for example, test Solid OIDC issuer. Then it provides a conformance report explaining which tests are passing and which are failing. It also provides MUST / MAY level.
+  * ...: It would be nice to get results as RDF. All the tests are described as RDF already.
+  * ...: Pete had suggestion, I use comment now, but spec vocab (currently on branch not published) has property called statement.
+  * ...: There are some things hard coded; for example, using specific crypt algorithms. CSS team has run it against their server; it was only failing very recent change to the spec introducing WebID scope.
+  * ...: Some UI design could happen here, for now it is a decent start.
+* eP: does it test interaction of OP with a client, or also with RS?
+* AC: currently it tests OP client interaction
+  * ...: it does test some OIDC interaction, well-known, public key, etc.
+  * ...: I'm only testing ID Token, anticipating changes coming next
+* eP: noticing annotations, there is nothing stopping us using JSON-LD in the spec document (rather than Turtle). What is the process of extracting the 'statement'.
+* AC: the current test suit has no dependencies. I'm pulling descriptions as JSON, since they are in compacted JSON-LD.
+  * ...: I have multiple suites. The test is just few lines of code. We pass identifier for relevant test, and whether it passed, plus optional message.
+  * ...: In data directory, I can describe new test or requirement, and reference it from test suite.
+  * ...: I'm performing some reflection to get methods of the class, so if you add a method to the class, it automatically runs.
+* PE: When we interpret the spec, you can see requirement level and requirement subject. If you do that in turtle fragments you would have duplication.
+  * ...: Is you report function going directly to HTML? I assume you can start outputting EARL.
+* AC: I currently do some DOM manipulation, but we could output the data in any other way.
+* PE: ??? [completely missed comment should probably be deleted, unless @edwardsph remembers what they said here]
```

Comment:
```suggestion
```

We can wait another day before deleting it, in case @edwardsph can recall what is missing here.

Reply:
Sorry - I can't remember what this was related to but there is nothing obviously missing so I'm happy for that to be deleted.